src/javascript/app/pages/resources/metatrader/download.ui.js

```diff
@@ -0,0 +1,54 @@
+import OSDetect from '../../../../../templates/_common/os_detect';
+
+const toggleDownloadPage = target => {
+    document.querySelectorAll('.download-block').forEach(block => {
+        if (block.getAttribute('id') === target) {
+            block.classList.add('visible');
```

Comment:
instead of `.classList.add('visible')` and `.classList.remove('invisible')` you can use `.setVisibility(0)` and `.setVisibility(1)` respectively

Reply:
The only reason behind using `visible` class is to create a possibility to hook on visible items to do transitions and stuff. There is no other logic behind this. 🤔 

I was looking for `.setVisibility()` API in MDN 😃. I did find it on our very own `plugins.js`. Yeah that's a good shortcut. I'll use it.